server/data/helpers/classesHelper.js

```diff
@@ -18,8 +18,8 @@ module.exports = {
         'teachers.lastname as t_last',
         'teachers.email as t_email'
       )
-      .join('classes', 'teachers.id', 'classes.teacher_id')
-      .where('classes.id', id);
+      .join('teachers_classes', 'teachers.id', 'teachers_classes.teacher_id')
+      .where('teachers_classes.class_id', id);
 
```

Comment:
Did you add the many to many table for teachers_classes?  I wasn't sure if we were going to let one class have multiple teachers or not. I think it makes sense if classes are actually just sections kind of like Lambda sections can have multiple teachers.

Reply:
Oh, my bad! It looks like I was still using my dev db, and I guess I forgot that we had removed teachers_classes in the new schema.

Kelly, can we hold off on merging this one? I will alter the tests so they work with the new schema.